wallet/encrypter/encrypter_test.go

```diff
@@ -66,3 +67,31 @@ func TestEncrypter(t *testing.T) {
 	_, err = enc.Decrypt(cipher, "invalid-password")
 	assert.ErrorIs(t, err, ErrInvalidPassword)
 }
+
+func TestEncrypterV3(t *testing.T) {
+	enc := &Encrypter{
```

Comment:
I think you can define two constructor functions for versioning to improve readability. For example:

```go
func NewEncryptorV2() *Encryptor {}
```

```go
func NewEncryptorV3() *Encryptor {}
```

Reply:
No, We only have one constructor that is `DefaultEncryptor`. 
the rest are initialized by JSON unmarshaling.